### PR TITLE
[opentelemetry/guide] Add guide for collecting host metadata via semantic conventions

### PR DESCRIPTION
**Pull request by mx-psi**
### What does this PR do? What is the motivation?

Adds guide for mapping OpenTelemetry semantic conventions into the infrastructure list

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by mx-psi** [comment]
This PR is reviewable, but please don't merge just yet, I want to sync with Datadog support before merging this :)

File: content/en/opentelemetry/guide/host_metadata.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+Lastly, add this processor to the `processors` list of all your pipelines.
+
+You also need to explicitly tag all your resources with a host-identifying attribute. This is done by default by the [recommended setup for host metrics][3].
+
+## What conventions are supported?
+
+The Datadog exporter supports both resource attribute-level semantic conventions as well as system metrics-level semantic conventions. Supported resource attribute semantic conventions are mainly under [the `host.` namespace][4] and [the `os.` namespace][8]. All supported system metrics-level semantic conventions are under [the `system.` namespace][5].
+
+The following general system conventions are supported:
```
Comment (liustanley) [inline content/en/opentelemetry/guide/host_metadata.md:55]: nit: Disregard if you think this is unnecessary, but this line could be made into a heading for navigation and clearer division of the tables. Same for lines 62 and 76.
```suggestion
### General system conventions
```

Comment (mx-psi) [reply content/en/opentelemetry/guide/host_metadata.md]: Looks better now, thanks for the suggestion

File: content/en/opentelemetry/guide/host_metadata.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+| Semantic convention | Type               | In-app field |
+|---------------------|--------------------|--------------|
+| [*Various*][10]     | Resource attribute | Hostname     |
```
[the remark this answers is not in the document]
Comment (mx-psi) [reply content/en/opentelemetry/guide/host_metadata.md]: Done!

**Review by liustanley — APPROVED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**